src/components/search/data/hooks.js

```diff
@@ -156,8 +116,8 @@ export const useDefaultSearchFilters = ({ enterpriseConfig, subscriptionPlan, of
       }
       return `enterprise_customer_uuids:${enterpriseConfig.uuid}`;
     },
-    [enterpriseConfig, subscriptionPlan, offerCatalogs, showAllCatalogs],
+    [enterpriseConfig, subscriptionPlan, offerCatalogs, activeRefinements[SHOW_ALL_NAME]],
```

Comment:
`activeRefinements?[SHOW_ALL_NAME]` so that this works even if the param is not present